java-filter-chain/src/main/java/org/wso2/mgw/filterchain/JWTValidator/JWTValidator.java

```diff
@@ -71,22 +76,32 @@ public String load(String s) throws Exception {
 
                     });
     //validate JWT token
-    public static boolean validateToken () {
+    public static CheckResponse validateToken (Map<String, String> headers) {
         boolean valid = false;
-        HashMap<String, String> request = new HashMap<String, String>();
-        request.put(JWTConstants.AUTHORIZATION, JWTConstants.JWT_TOKEN);
-        for (Map.Entry mapElement : request.entrySet()) {
-            String key = (String) mapElement.getKey();
-            if (key == JWTConstants.AUTHORIZATION) {
-                valid = HandleJWT(request);
-                break;
-            }
+        CheckResponse response;
+
+        String token = headers.get(JWTConstants.AUTHORIZATION);
+        //Map<String, String> request = new HashMap<String, String>();
+        //request.put(JWTConstants.AUTHORIZATION, JWTConstants.JWT_TOKEN);
```

Comment:
Shall we remove the commented code lines?

Reply:
This was removed